tests/requirements.txt

```diff
@@ -3,6 +3,6 @@ nose
 parameterized
 six
 coverage
-flake8
+flake8<3.8
```

Comment:
a workaround until this is merged and released: https://github.com/tholo/pytest-flake8/pull/68

Reply:
Maybe link a comment with a link to that issue, just in case?

Reply:
fixed now :slightly_smiling_face: